galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.eclipse.runtime.manager/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+-snapshot: ${tstamp}
```

Comment:
The projects are still called dev.galasa.eclipse.runtime,    needs to be dev.galasa.eclipseruntime